tasks.py

```diff
@@ -53,3 +75,55 @@ def integrations_changed(ctx):
         if line:
             checks.add(line.split('/')[0])
     return checks
+
+
+@task(help={
+    'package': 'The package to upgrade throughout the integrations',
+    'version': 'The version of the package to pin',
+    'verbose': 'Whether or not to produce output',
+})
+def upgrade(ctx, package, version, verbose=False):
```

Comment:
Can you change these to flag arguments instead of positional ones (you can just exit with an error if there are none). This is a workaround to a bug in invoke that doesn't allow you to call help after the invoke task:

```
~/d/integrations-core  ofek/add-upgrade-task ◼ ❯❯❯ invoke upgrade -h
'upgrade' did not receive all required positional arguments!
~/d/integrations-core  ofek/add-upgrade-task ◼ ❯❯❯ invoke -h upgrade
Usage: inv[oke] [--core-opts] upgrade [--options] [other tasks here ...]

Docstring:
  none

Options:
  -e, --verbose                 Whether or not to produce output
  -p STRING, --package=STRING   The package to upgrade throughout the integrations
  -v STRING, --version=STRING   The version of the package to pin
```